### ISSUE TITLE
[OAKTrust] Browse by Subject Category

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#208]
**URL:** https://oaktrust-dev.library.tamu.edu/browse/srsc
**WCAG 2.1 AA compliant? :** No
**Violation:** 2.1.1 Keyboard (A)
**Issues:**  Can't navigate tree structure
**Notes:** Can only select humanities and religion checkbox. Can't expand any branch beyond humanities and religion.
